src/Groups/GAPGroups.jl

```diff
@@ -182,8 +182,8 @@ end
 
 order(x::Union{GAPGroupElem, GAPGroup}) = order(fmpz, x)
 
-@gapwrap has_order(G::GAPGroup) = GAP.Globals.HasSize(G.X)
-@gapwrap set_order(G::GAPGroup, val::T) where T<:IntegerUnion = GAP.Globals.SetSize(G.X, GapObj(val))
+@gapwrap has_order(G::GAPGroup) = GAP.Globals.HasOrder(G.X)
+@gapwrap set_order(G::GAPGroup, val::T) where T<:IntegerUnion = GAP.Globals.SetOrder(G.X, GapObj(val))
```

Comment:
I do not understand this.
In GAP, `Size` and `Order` are independent operations, as well as their testers and setters.
If `HasSize( G )` is `true` then `HasOrder( G )` can be `false`, and calling `SetOrder` for a group does not set `Size` for it. Since `Size` is the attribute used for groups, `HasOrder` and `SetOrder` do not help.

Reply:
Oops, thanks for catching this. (It's from a separate commit that I meant to discard)